201-vmss-custom-script-windows/azuredeploy.parameters.us.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
there doesn't seem to be anything specific to USGov in this file - use these param value is azuredeploy.parameters.json and then remove this file.